addons/crm/test/lead2opportunity2win.yml

```diff
@@ -48,19 +48,6 @@
   !python {model: crm.lead}: |
     self.message_post(cr, uid, [ref('crm_case_3')], subject='Test note', body='Détails envoyés par le client sur ​​le FAX pour la qualité')
 -
-  I win this opportunity
```

Comment:
Hello @mge-odoo 
That because this test case is works on case_default stage, and now case_default stage field is removed. So this case not longer useful.